internal/manifests/opampbridge/deployment.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opampbridge
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/open-telemetry/opentelemetry-operator/internal/manifests"
+	"github.com/open-telemetry/opentelemetry-operator/internal/naming"
+)
+
+// Deployment builds the deployment for the given instance.
+func Deployment(params manifests.Params) *appsv1.Deployment {
+	name := naming.OpAMPBridge(params.OpAMPBridge.Name)
+	labels := Labels(params.OpAMPBridge, name, params.Config.LabelsFilter())
+
+	return &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: params.OpAMPBridge.Namespace,
+			Labels:    labels,
```

Comment:
Annotations are not propagated? The CM propagates them.

Reply:
> Annotations are not propagated? The CM propagates them.

Fixed.